_platforms/en/datigov.md

```diff
@@ -21,6 +21,25 @@ socials:
     name: youtube
     icon: youtube
     link: //www.youtube.com/channel/UCHsqR2YhxeKgFT4rAKVcAAQ
+resources:
+    - Documentation:
+        - title: Architectural document for evolution of dati.gov.it
+          icon: file
+          url: https://docs.google.com/document/d/1dCfbpwmkl-U2kreykRY8YXJct6GOe-r_3qjism5wI2Y
+        - title: Technical guidelines for data catalogs
+          icon: file
+          url: https://docs.italia.it/italia/daf/linee-guida-cataloghi-dati-dcat-ap-it/
+          desc: Technical guidelines for data catalogs including DCAT-AP_IT spec
+    - Source code:
+        - title: dati.gov.it repository
+          icon: github
+          url: https://github.com/FormezPA/dkan
```

Comment:
I'm not sure that's the official one, let's wait for an answer before publishing.